oojs.js

```diff
@@ -1,3 +1,24 @@
+class Playlist {
+  constructor(title, ){
+    this.title = title;
+    this.song = [];
+    this.favorite = 0;
+  }
+}
+addSong(song){
+  this.song.push(song);
+  console.log(this.song);
+}
+
+const playlistName = new Playlist("playlistName")
+
+class album extends Playlist{
+   constructor(artist){
+    this.artist = artist;
+  }}
```

Comment:
Because this inherits from the Album class, you need to call `super()` inside of the constructor and make sure to pass in the correct arguments.